api-specificatie/openapi.yaml

```diff
@@ -1054,519 +1003,215 @@ components:
           $ref: '#/components/schemas/NEN3610ID'
         adresseerbaarObjectIdentificatie:
           $ref: '#/components/schemas/NEN3610ID'
-        pandIdentificatie:
+        pandIdentificaties:
           type: array
           items:
             $ref: '#/components/schemas/NEN3610ID'
           minItems: 1
-        nevenadres:
-          description: 'Boolean die aangeeft of dit adres een nevenadres is. Als deze boolean niet wordt meegeleverd (of als deze false is) dan is dit een Hoofdadres.'
+        indicatieNevenadres:
+          description: 'Een aanduiding die aangeeft of dit adres een nevenadres is van een adresseerbaar object. Als deze boolean niet wordt meegeleverd (of als deze false is) dan is dit een Hoofdadres.'
           type: boolean
-        geconstateerd:
-          description: 'Een aanduiding waarmee kan worden aangegeven dat een object in de registratie is opgenomen als gevolg van een feitelijke constatering, zonder dat er op het moment van opname sprake was van een regulier brondocument voor deze opname.'
+        indicatieGeconstateerd:
+          description: 'Een aanduiding waarmee kan worden aangegeven dat één of meerdere gerelateerde object(en) in de registratie is/zijn opgenomen als gevolg van een feitelijke constatering, zonder dat er op het moment van opname sprake was van een regulier brondocument voor deze opname.'
           type: boolean
-        indicatieInOnderzoek:
-          description: 'Een indicatie dat één of meerdere attributen van objecten gerelateerd aan een adres in onderzoek is/zijn.'
+        indicatieMogelijkOnjuist:
+          description: 'Een aanduiding dat één of meerdere attributen van objecten gerelateerd aan een adres in onderzoek of mogelijk onjuist is/zijn.'
           type: boolean
-
+    AdresHal:
+      type: object
```

Comment:
'type: object' geeft hier geen foutmelding, maar hoort hier niet. Volgens mij moet het zijn:

      allOf:
      - $ref: '#/components/schemas/AdresHalBasis'
      - object:
        properties:
          _embedded:
             $ref: '#/components/schemas/Adres_embedded'

Zie: Models with Composition @ https://swagger.io/specification/

Reply:
object is een foutje geweest. Die regel moet weg . Na de Ref moet direct - properties komen. 
Aangepast